SCRAPERS-LIST.md

```diff
@@ -862,15 +862,15 @@ naughtyamerica.com|NaughtyAmerica.yml|:heavy_check_mark:|:x:|:x:|:x:|-|-
 naughtyamericavr.com|NaughtyAmericaVR.yml|:heavy_check_mark:|:x:|:x:|:x:|-|VR
 newsensations.com/tour_ns/|NewSensationsMain.yml|:heavy_check_mark:|:x:|:heavy_check_mark:|:x:|-|-
 newsensations.com/tour_rs/|NewSensationsNetworkSites.yml|:heavy_check_mark:|:x:|:x:|:x:|-|-
-nextdoorbuddies.com|GammaEntertainment.yml|:heavy_check_mark:|:x:|:x:|:x:|-|Gay
-nextdoorcasting.com|GammaEntertainment.yml|:heavy_check_mark:|:x:|:x:|:x:|-|Gay
-nextdoorebony.com|GammaEntertainment.yml|:heavy_check_mark:|:x:|:x:|:x:|-|Gay
-nextdoorhookups.com|GammaEntertainment.yml|:heavy_check_mark:|:x:|:x:|:x:|-|Gay
-nextdoormale.com|GammaEntertainment.yml|:heavy_check_mark:|:x:|:x:|:x:|-|Gay
-nextdoorraw.com|GammaEntertainment.yml|:heavy_check_mark:|:x:|:x:|:x:|-|Gay
-nextdoorstudios.com|GammaEntertainment.yml|:heavy_check_mark:|:x:|:x:|:x:|-|Gay
-nextdoortaboo.com|GammaEntertainment.yml|:heavy_check_mark:|:x:|:x:|:x:|-|Gay
-nextdoortwink.com|GammaEntertainment.yml|:heavy_check_mark:|:x:|:x:|:x:|-|Gay
+nextdoorbuddies.com|Algolia_NextdoorStudios.yml|:heavy_check_mark:|:x:|:x:|:x:|Python|Gay
+nextdoorcasting.com|Algolia_NextdoorStudios.yml|:heavy_check_mark:|:x:|:x:|:x:|Python|Gay
+nextdoorebony.com|Algolia_NextdoorStudios.yml|:heavy_check_mark:|:x:|:x:|:x:|Python|Gay
+nextdoorhookups.com|Algolia_NextdoorStudios.yml|:heavy_check_mark:|:x:|:x:|:x:|Python|Gay
+nextdoormale.com|Algolia_NextdoorStudios.yml|:heavy_check_mark:|:x:|:x:|:x:|Python|Gay
```

Comment:
These 2 sites havent migrated to the api yet?
We need to keep them in the Gamma scraper